### PR TITLE
Fixed the manager option in the agent deployment section

### PR DESCRIPTION
**Pull request by yenienserrano**
### Description
Added options if you are in manager mode instead of cluster mode
 
### Issues Resolved
- [#4938](https://github.com/wazuh/wazuh-kibana-app/issues/4938)

### Evidence

[Screencast from 12-12-2022 05:15:44 PM.webm](https://user-images.githubusercontent.com/63758389/207145751-51c45083-80a4-4eef-8c0b-ee769e40964b.webm)

### Environment

- Have an environment in manager mode

### Test

- Enter wazuh and in the module overview the toast should not appear with error 3013.

### Check List
- [ ] All tests pass
  - [ ] `yarn test:jest`
- [ ] New functionality includes testing.
- [x] New functionality has been documented.
- [x] Update [CHANGELOG.md](./../CHANGELOG.md)
- [x] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: public/controllers/agent/components/register-agent-service.ts [hunk 1]
```diff
@@ -112,11 +143,22 @@ export const getNodeIPs = async (): Promise<any> => {
   return await WzRequest.apiReq('GET', '/cluster/nodes', {});
 };
 
+export const getManagerNode = async (): Promise<any> => {
+  const managerNode = await WzRequest.apiReq('GET', '/manager/api/config', {});
+  return managerNode.data.data.affected_items.map(item => ({
```
Comment (asteriscos) [inline public/controllers/agent/components/register-agent-service.ts:148]: If the API response is an error, is it handled somehow?

**Review by asteriscos — CHANGES_REQUESTED** [review]
CR: See comments
Test: :x: 

<details>

<summary> Evidence </summary>

Peek 2022-12-19 18-46


</details>

**Comment by yenienserrano** [comment]
> CR: See comments Test: x
> 
> Evidence
> Peek 2022-12-19 18-46


I was trying to reproduce this error, but I could not

File: public/controllers/agent/components/register-agent-service.ts [hunk 2]
```diff
@@ -17,22 +17,47 @@ type RemoteConfig = {
   haveSecureConnection: boolean | null;
 };
 
+/**
+ * Get the cluster status
+ */
+export const clusterStatusResponse = async (): Promise<boolean> => {
+  const clusterStatus = await WzRequest.apiReq('GET', '/cluster/status', {});
```
Comment (asteriscos) [inline public/controllers/agent/components/register-agent-service.ts:24]: Should this request handle possible API errors?

**Review by Machi3mfl — APPROVED** [review]
CR: :heavy_check_mark: 
Test: :heavy_check_mark:

File: public/controllers/agent/components/register-agent-service.ts (hunk 2 above)
Comment (yenienserrano) [reply public/controllers/agent/components/register-agent-service.ts:24]: The errors are controlled in this function

https://github.com/wazuh/wazuh-kibana-app/blob/4.4-7.16/public/controllers/agent/register-agent/steps/server-address.tsx#L39-L63

**Review by asteriscos — APPROVED** [review]

**Pull request merged**